### ISSUE TITLE
Cyborgs cannot deconstruct tables

### ISSUE THREAD

**Issue by Scilianty** [Skyrat-SS13/Skyrat-tg#2001]
Reporting client version: 513.1536

## Round ID:
[96](https://scrubby.melonmesa.com/round/96)

## Testmerges:
- [[TESTMERGE] Makes icon2html not work](https://github.com/Skyrat-SS13/Skyrat-tg/pull/948)
- [[TESTMERGE-ONLY] [DNM] Stack traces on some things](https://github.com/Skyrat-SS13/Skyrat-tg/pull/1848)
- [[DNM] Cryo on core maps [TESTMERGE ONLY]](https://github.com/Skyrat-SS13/Skyrat-tg/pull/1783)
- [[WIP] Combat rework - less lethality, less hardstuns, nonlethal takedowns. Focus on combat ineffectivity](https://github.com/Skyrat-SS13/Skyrat-tg/pull/1285)

## Reproduction:
simply put, the engineering cyborg cannot use it's screwdriver or wrench to deconstruct any form of table, regardless of what's it's made of. Additionally, when attempting to deconstruct reinforced tables, the first step of welding does not work, however it will use fuel from the welder, despite having no effect.

**Comment by Scilianty** [comment]
Error making this issue, you must be on harm intent to deconstruct.
